molecule/default/roles/test_glauth/tasks/verify.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Detect glauth.toml file
+  ansible.builtin.stat:
+    path: /etc/glauth/glauth.toml
+  register: glauth_toml
+
+- name: Assert ownership and permissions are correct
+  ansible.builtin.assert:
+    that:
+      - glauth_toml.stat.exists
+      - glauth_toml.stat.mode == '0640'
+      - glauth_toml.stat.pw_name == 'root'
+      - glauth_toml.stat.gr_name == 'glauth'
+
+- name: Check syntax of glauth config
+  ansible.builtin.command:
+    cmd: /usr/bin/glauth -c {{ glauth_toml.stat.path }} --check-config
```

Comment:
the path is hardcoded in line 4, so not sure this is useful

Reply:
I chose this style in order to hard-code it only once. I can use the path everywhere if you prefer.

Reply:
not sure it is an improvement, tbh.. there might be other cases where you won't have the stat result.. I'd say either hardcode everywhere or set a variable somewhere

Reply:
Changed to hard-coding everywhere!